apps/courses/src/components/Footer.tsx

```diff
@@ -64,26 +66,10 @@ export default (props) => (
             >
               <Flex direction="column">
                 {group.links.map((item, i) => {
-                  const isExternal =
-                    item.link.includes('http://') ||
-                    item.link.includes('https://');
-
-                  const linkProps = isExternal
-                    ? {
-                        target: '_blank',
-                        rel: 'noopener noreferrer',
-                        href: item.link,
-                        as: 'a',
-                      }
-                    : {
-                        to: item.link,
-                        as: RRDLink,
-                      };
-
                   return (
```

Comment:
In this case, I'd prefer we skip the return statement and use parentheses as the return:

**You**
```js
thing.map(i => {
  return <Blah />;
})
```

**What I'd like**
```js
thing.map(i => (
  <Blah />;
))
```